src/playbacks/no_flash/public/style.scss

```diff
@@ -0,0 +1,25 @@
+[data-no-flash] {
+  z-index: 1000;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  text-align: center;
+}
+
+[data-no-flash] p[data-no-flash-msg] {
+  position: absolute;
+  font-size: 25px;
+  top: 40%;
+  color: white;
+}
+
+[data-no-flash] p[data-no-flash-msg] a {
+  color: white;
+}
+
+[data-no-flash] canvas[data-no-flash-canvas] {
+  background-color: #777;
```

Comment:
Color literals like `#777` should only be used in variable declarations; they should be referred to via variable everywhere else.